src/components/DroughtHistory.vue

```diff
@@ -6,7 +6,7 @@
       </div>
       <div id="intro-container">
         <p>
-          The U.S. has experienced thousands of droughts—periods of drier-than-normal conditions that cause water-related problems for humans and ecosystems (see <a
+          The U.S. has experienced thousands of droughts that can cause water-related problems for humans and ecosystems (see <a
             href="https://labs.waterdata.usgs.gov/visualizations/what-is-drought/index.html#/"
```

Comment:
I don't think this is a reference for that information. I would use this to reference how we can measure drought events at a gage